### ISSUE TITLE
Underground Trigger json not loading

### ISSUE THREAD

**Issue by Cygnus-Altair** [Arkensor/DayZCommunityOfflineMode#356]
While I see the missions folder has the cfgundergroundtrigger.json file, when I copy in the version from my test server it does not work.
It works on my test server, so I assume the functionality has not been added to the OfflineMode.  Can this be added?

**Comment by Arkensor** [comment]
Yes, this functionality is currently not present. I will be adding it at some point, but most likely not within 2022.
